datadog_checks_dev/datadog_checks/dev/tooling/commands/manifest.py

```diff
@@ -76,12 +69,15 @@ def verify(fix, include_extras):
         manifest_file = os.path.join(root, check_name, 'manifest.json')
 
         if file_exists(manifest_file):
-            display_queue = [(echo_info, '{} ->'.format(check_name))]
+            echo_info('Checking {}/manifest.json...'.format(check_name), nl=False)
```

Comment:
This needs to be behind a verbose flag; the current behavior is intentional. We don't want to output 100 lines by default of `everything is ok`. It will also muddle the output when there are errors to quickly see what's wrong, even with colors.

Reply:
May also be OK to provide a count of the OK checks, like `85 OK checks and 2 Failed ones` with more info on the failed ones like you have?